contrib/opencensus-ext-ocagent/opencensus/ext/ocagent/stats_exporter/__init__.py

```diff
@@ -0,0 +1,246 @@
+# Copyright 2018, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from google.api_core import bidi
+from opencensus.ext.ocagent import utils
+from opencensus.metrics import transport
+from opencensus.metrics.export import metric_descriptor
+from opencensus.metrics.export import value
+from opencensus.proto.agent.metrics.v1 import metrics_service_pb2
+from opencensus.proto.agent.metrics.v1 import metrics_service_pb2_grpc
+from opencensus.proto.metrics.v1 import metrics_pb2
+from opencensus.stats import stats
+import grpc
+
+
+class StatsExporter(object):
```

Comment:
Reminder that we'll need to rename these `MetricsExporter` eventually.

Reply:
Ack.